erasure_code/aarch64/gf_4vect_dot_prod_neon.S

```diff
@@ -26,11 +26,18 @@
   (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
   OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 **********************************************************************/
+#if defined(__MACH__)
```

Comment:
Is it Xcode or Clang request ? 
I think __clang__ or __xcode__(?) might be better macros

Reply:
sorry, actually it is from https://stackoverflow.com/a/10145363 . I'll update when I find better info source.

Reply:
https://news.ycombinator.com/item?id=20143037 points http://math-atlas.sourceforge.net/devel/assembly/MachORuntime.pdf

```
A symbol is a generic representation of the location of a function, data variable, or constant in an
executable file. References to functions and data in a program are references to symbols. To refer to
a symbol when using the dynamic linking routines, you usually pass the name of the symbol, although
some functions also accept a number representing the ordering of the symbol in the executable file.
The name of a symbol representing a function that conforms to standard C calling conventions is the
name of the function with an underscore prefix. Thus, the name of the symbol representing the
function main would be _main.
```

Reply:
I mean the MACRO . It looks it should be `#if defined(__WIN32__) || defined(__APPLE__)`

Reply:
It turned out we can use __USER_LABEL_PREFIX__. Fixed.

There are several implementations, but this time I refered http://svn.python.org/projects/external/ctypes-0.9.9.4/source/libffi/src/arm/sysv.S, which is MIT license.